packages/flutter_app_badger/example/integration_test/flutter_app_badger_test.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:flutter_app_badger/flutter_app_badger.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('get supported', (tester) async {
+    expect(await FlutterAppBadger.isAppBadgeSupported(), true);
+  });
+
+  testWidgets('update badge count', (tester) async {
+    await FlutterAppBadger.updateBadgeCount(1);
+  });
+
+  testWidgets('remove badge count', (tester) async {
+    await FlutterAppBadger.removeBadge();
+  });
```

Comment:
There are no `expect` in these tests.

Reply:
The original code (`FlutterAppBadger` class) does not provide a separate exception or return type.
I think this test is a test that runs without exception.
In this case, do I need to handle it separately?
Or are you talking about deleting this TC?

Reply:
Without any `expect` in a test, we can't judge anything properly.
You can think of a few ways to this...but I found followings, how about it?

```dart
await expectLater(FlutterAppBadger.updateBadgeCount(1), completes);
```

Reply:
Thank you for your advice. I updated code